src/main/java/com/stripe/model/PaymentLink.java

```diff
@@ -119,13 +135,29 @@ public class PaymentLink extends ApiResource implements HasId, MetadataStore<Pay
   @SerializedName("shipping_address_collection")
   ShippingAddressCollection shippingAddressCollection;
 
+  /** The shipping rate options applied to the session. */
+  @SerializedName("shipping_options")
+  List<PaymentLink.ShippingOption> shippingOptions;
```

Comment:
Why is there a prefix here? Isn't it the default to use the local inner class?

Reply:
The generator apparently derps when the reference is inside a type parameter. Harmless but worth fixing because it makes the eyeball test more reliable.